pkgs/by-name/gr/gradle-dependency-tree-diff/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gradle,
+  makeBinaryWrapper,
+  jre_headless,
+  zulu11,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gradle-dependency-tree-diff";
+  version = "1.2.1";
+  src = fetchFromGitHub {
+    owner = "JakeWharton";
+    repo = "dependency-tree-diff";
+    tag = "${finalAttrs.version}";
+    hash = "sha256-7ObmZygzSp7aAnqsJuMcPk+I3z993kjHCJMug3JkONg=";
+  };
+
+  nativeBuildInputs = [
+    gradle
+    makeBinaryWrapper
+  ];
+
+  mitmCache = gradle.fetchDeps {
+    inherit (finalAttrs) pname;
+    data = ./deps.json;
+  };
+  __darwinAllowLocalNetworking = true;
```

Comment:
why is this required?

Reply:
This is from the Gradle packaging tutorial in tree:
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/gradle.section.md#building-a-gradle-package-building-a-gradle-package

> ```nix
>   ...
>   # this is required for using mitm-cache on Darwin
>   __darwinAllowLocalNetworking = true;
>   ...
> ```